packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -38,7 +38,7 @@ import * as Swagger2OpenAPI from 'swagger2openapi'
 import * as OASValidator from 'oas-validator'
 import debug from 'debug'
 import { handleWarning } from './utils'
-import { parseType } from 'graphql'
+import * as pluralize from 'pluralize'
```

Comment:
Are we making an assumption that the API is in english? Perhaps we should expose an API to add pluralization rules from a JSON file

Reply:
Thank you for that comment! I think it was the implicit assumption so far that the API is in English, yes. But as you point out, there could be the need for pluralizing names for APIs in other languages.

Thinking about an option, we would propose to allow developers to optionally add their own `pluralize` function, which takes as input a string and is expected to output a (pluralized) string. The default `pluralize` library would be used if that option is not provided.

Reply:
A side comment: the string passed to `pluralize` is not necessarily a single word - it is basically the sanitized version of a path component, which could be anything, e.g., a combination of words.

Reply:
@omarchehab98 We wanna go ahead in the next days and merge this PR. We are thinking to not allow passing a custom `pluralize` for now, as we assume most interfaces to be in English (which is made likely by the constraints that the GraphQL spec has for names).

However, if we hear that people have specific needs for custom pluralize functions, we would reconsider. Do you actually have the need for a custom pluralize function?

Reply:
Some teams name things in other languages other than English, you can go without a custom pluralize function until someone opens an issue about it.

Reply:
Thanks for your response! I think this will be our way forward for now: not add the option until specific requirements are stated.